portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,264 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
jQuery is a very heavy library just for two lines of code. Consider using just vanilla JavaScript instead.